CODE_OF_CONDUCT.md

```diff
@@ -1,42 +1,136 @@
 # Bundler Code of Conduct
 
-The Bundler project strongly values contributors from anywhere, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, ethnicity, age, religion, or nationality. As a result, the Bundler team has agreed to and enforces this code of conduct in order to provide a harassment-free experience for everyone who participates in the development of Bundler.
+## When Something Happens
 
-### Summary
+If you see a Code of Conduct violation, follow these steps:
 
-Harassment in code and discussion or violation of physical boundaries is completely unacceptable anywhere in the Bundler project’s codebases, issue trackers, chat rooms, mailing lists, meetups, and any other events. Violators will be warned and then blocked or banned by the core team at or before the 3rd violation.
+1.  Let the person know that what they did is not appropriate and ask them to stop and/or edit their message(s).
+2.  That person should immediately stop the behavior and correct the issue.
+3.  If this doesn't happen, or if you're uncomfortable speaking up, [contact admins](#contacting-admins).
+4.  As soon as available, an admin will join, identify themselves, and take [further action (see below)](#further-enforcement), starting with a warning, then temporary deactivation, then long-term deactivation.
 
-### In detail
+When reporting, please include any relevant details, links, screenshots, context, or other information that may be used to better understand and resolve the situation.
 
-Harassment includes offensive verbal comments related to level of experience, gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, ethnicity, age, religion, nationality, the use of sexualized language or imagery, deliberate intimidation, stalking, sustained disruption, and unwelcome sexual attention.
+**The admin team will prioritize the well-being and comfort of the recipients of the violation over the comfort of the violator.** See [some examples below](#enforcement-examples).
 
-Individuals asked to stop any harassing behavior are expected to comply immediately.
+## Our Pledge
 
-Maintainers, including the core team, are also subject to the anti-harassment policy.
+In the interest of fostering an open and welcoming environment, we as members of the package.community community pledge to making participation in our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, technical preferences, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
This says `package.community`, is that intentional?

Reply:
No, blah. Thanks for catching that.